Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_can.c

```diff
@@ -211,6 +211,7 @@
 
 /* Includes ------------------------------------------------------------------*/
 #include "stm32f4xx_hal.h"
+#include "main.h"
```

Comment:
Is there a reason you had to include this. I thought we want to avoid changes to library files as much as possible.